README.md

```diff
@@ -41,3 +41,14 @@ Match the layout as close as you can, using free and open fonts and graphics.
 
 - does not have tablet layout
 - uses a background image that is not included, you will have to find something similar (subtlepatterns.com)
+
+#### Stretch Goals
+
+Once you have finished with all of the requirements outlined above, you are encouraged to work on these additional stretch goals to improve the quality and usablity of your application
+
+- Make friendly error messages for users attempting to login using [connect-flash](https://github.com/jaredhanson/connect-flash)
+- Make the site beautiful with SCSS, no external libraries are allowed, be sure to use vanilla CSS and JavaScript
```

Comment:
Might be able to word this a little more clearer. Might be confusing saying to use SCSS, but only use vanilla CSS and JavaScript, since SCSS is above vanilla CSS.